src/components/Auth/Details.tsx

```diff
@@ -49,6 +51,8 @@ const Details: FC = () => {
   `)
   const {
     data: { lessons, weeks, habits },
+    status,
+    isLoading,
```

Comment:
This is most likely unused variable => our linter no likey

Reply:
`status` is used for the loader. `isLoading` is useless. Removed it.